src/asmcnc/comms/serial_connection.py

```diff
@@ -845,6 +855,49 @@ def process_grbl_push(self, message):
                     self.feed_rate = feed_speed[0]
                     self.spindle_speed = feed_speed[1]
 
+                # TEMPERATURES
+                elif part.startswith('TC:'):
+                    temps = part[3:].split(',')
+
+                    try: 
+                        float(temps[0])
+                        float(temps[1])
+                    except:
+                        log("ERROR status parse: Temperature invalid: " + message)
+                        return
+
+                    self.pcb_temp = float(temps[0])
+                    self.motor_driver_temp = float(temps[1])
+
+                    try:
+                        float(temps[2])
+                        self.transistor_heatsink_temp = float(temps[2])
+
+                    except IndexError:
+                        pass
+
+                    except Exception as E:
```

Comment:
specify exception type?

Reply:
plus no reference to E is ever made as far as I can tell

Reply:
good catch - was using it for debugging, will fix 😁